src/DIRAC/Resources/Cloud/CloudEndpoint.py

```diff
@@ -237,7 +237,7 @@ def createInstance(self, instanceID="", createPublicIP=True):
         createNodeDict["image"] = image
 
         # Get the flavor object
-        if "FlavorID" in self.parameters and "FlavorName" not in self.parameters:
+        if "FlavorID" not in self.parameters and "FlavorName" in self.parameters:
```

Comment:
Would it be more readable to re-order the if conditions to be like this:
```python
        if "FlavorID" in self.parameters:
            flavor = self.__driver.ex_get_size(self.parameters["FlavorID"])
        elif "FlavorName" in self.parameters:
            result = self.__getFlavorByName(self.parameters["FlavorName"])
            if not result["OK"]:
                return result
            flavor = result["Value"]
        else:
            return S_ERROR("No flavor specified")
```

Reply:
Yes, it is better this way. Need another pull request?